contact.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<!-- Bootstrap CSS File -->
+<link href="lib/bootstrap/css/bootstrap.min.css" rel="stylesheet">
+
+<!-- Icon CSS File -->
+<link href="lib/ionicons/css/ionicons.min.css" rel="stylesheet">
+
+<!-- Google Fonts -->
+<link href="https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700" rel="stylesheet">
+
+<!-- CSS global -->
+<link rel="stylesheet" href="css/global.css">
+</head>
+<body>
+
+  <!-- HEADER and NAVBAR -->
+<div class="header">
+  <h2>
+    <img src="images/IESEG-Logo.jpg" height="40px" alt="Logo IESEG">
+    Location
+  </h2>
+  <div class="topnav">
+    <a class="active" href="index.html">Home</a>
+    <div class="subnav">
+      <button class="subnavbtn">Join<i class="fa fa-caret-down"></i></button>
+      <div class="subnav-content">
+        <a href="login.html">Login</a>
+        <a href="signup.html">Sign-Up</a>
+      </div>
+    </div>
+    <a href="profile.html">Profile</a>
+    <a href="about.html">About</a>
+    <a href="contact.html">Contact</a>
+    <input type="text" placeholder="Search..">
+  </div>
+</div>
+ <!-- Appartement list -->
+<div class="row" style="margin:0px;">
+  <div class="leftcolumn">
+     <!-- Appartement list 1-->
+ <div class="card wrap">
+  <div class="row no-gutters">
+    <h2>Contactez-nous</h2><br>
+    <form action="#" method="post">
+        <input type="text" placeholder="Votre adresse mail" name="email"><br>
+        <input type="text" placeholder="Objet" name="objet"><br>
+        <textarea name="message" cols="100" rows="10" placeholder="Votre message"></textarea><br>
+        <input type="submit" value="Envoyer">
+    </form>
+</div>
+</div>
+      </div>
+
+     <!-- Right colum w/ most popular location -->
```

Comment:
why there is the most popular location in the about page ?

Reply:
and in the other pull request Markus already create an about page

Reply:
Oh yes, my bad, i didn't select the correct file !